src/images/sea-map/Person_6.svg

```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 326.75 639.25">
```

Comment:
Why is this one not minified?

Reply:
I probably pressed save in here, it's just formatted the lines out. I'm happy to leave as I'll be splitting the SVGs into layers next.